### PR TITLE
[framework] Remove redundant path segments (part 4)

### PR DESCRIPTION
**Pull request by nemrism**
Remove redundant path segments in relative links inside /framework/configure-apps/file-schema/wcf/* files

Contributes to #10653

### CODE REVIEW

File: docs/framework/configure-apps/file-schema/wcf/behavior-of-servicebehaviors.md [hunk 1]
```diff
@@ -4,10 +4,10 @@ ms.date: "03/30/2017"
...
   
 > [!NOTE]
-> Behavior elements specific to Windows Workflow activities, such as the [\<sendMessageChannelCache>](../../../../../docs/framework/configure-apps/file-schema/windows-workflow-foundation/sendmessagechannelcache.md) element, are documented in the [\<behavior> of \<serviceBehaviors>](../../../../../docs/framework/configure-apps/file-schema/windows-workflow-foundation/behavior-of-servicebehaviors-of-workflow.md) page.  
+>  Behavior elements specific to Windows Workflow activities, such as the [\<sendMessageChannelCache>](../windows-workflow-foundation/sendmessagechannelcache.md) element, are documented in the [\<behavior> of \<serviceBehaviors>](../windows-workflow-foundation/behavior-of-servicebehaviors-of-workflow.md) page.
```
Comment (mairaw) [inline docs/framework/configure-apps/file-schema/wcf/behavior-of-servicebehaviors.md]:
```suggestion
> Behavior elements specific to Windows Workflow activities, such as the [\<sendMessageChannelCache>](../windows-workflow-foundation/sendmessagechannelcache.md) element, are documented in the [\<behavior> of \<serviceBehaviors>](../windows-workflow-foundation/behavior-of-servicebehaviors-of-workflow.md) page.  
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Changes look great @nemrism! Just found a space being introduced that we should remove it. 

It would be great if these PRs could be 100 files or less. GitHub stops loading the diff after that, so I can't see the entire diff unless I click on each file.

**Comment by nemrism** [comment]
Sorry about the number of files... I'm doing that with search and replace by subfolders.
Sometimes it involve more than 100 files and I didn't notice the diff limitation on Github website. I'm using the desktop software to review changes.

But I'll try to break the commits in different PRs to limit that and ease your reviews.

**Comment by mairaw** [comment]
Yes, search and replace is good. After you finish your changes, you can break down the files into multiple working branches. Let me know if I can help!

**Review by mairaw — APPROVED** [review]
Thanks for making the small change.

**Pull request merged**